### ISSUE TITLE
how to get 'cx', 'cy' normalized coordinates on ComputeLoss ?

### ISSUE THREAD

**Issue by GiUKim** [ultralytics/yolov5#10036]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

i want to get 'cx' 'cy' 0~1 normalized value such as Labeling in "utils/loss.py" ComputeLoss().
but the values from "pxy", "pwh" seems like anchors grid coordinate.

```
pxy = pxy.sigmoid() * 2 - 0.5
pwh = (pwh.sigmoid() * 2) ** 2 * anchors[i]
pbox = torch.cat((pxy, pwh), 1)  # predicted box
```

how to get this value(0 ~1 normalized coordinate) ?

**Comment by glenn-jocher** [comment]
@GiUKim to obtain the 'cx' and 'cy' normalized coordinates in ComputeLoss, you can calculate them from the cell grid offsets 'pxy' by applying the sigmoid function and then multiplying by the grid size, typically the image size. Here's an example of how to do it:

```python
# Calculate normalized 'cx' and 'cy'
pxy_normalized = (pxy.sigmoid() * 2 - 0.5 + grid_offset) / grid_size
```

For more details, you can refer to the YOLOv5 documentation at https://docs.ultralytics.com/yolov5/. Let me know if you need further assistance!
